flink-runtime/src/main/resources/META-INF/services/org.apache.flink.runtime.security.token.DelegationTokenProvider

```diff
@@ -14,3 +14,4 @@
 # limitations under the License.
 
 org.apache.flink.runtime.security.token.HadoopFSDelegationTokenProvider
+org.apache.flink.runtime.security.token.HBaseDelegationTokenProvider
```

Comment:
Hi, @gaborgsomogyi . Just a minor question.
Why `HBaseDelegationTokenProvider` is default here ?
I packaged flink-dist and used flink-1.17. 
If I don't remove this, an exception about lacking some hbase classes will be thrown.